primitives/keystore/src/lib.rs

```diff
@@ -0,0 +1,385 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2019-2020 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
```

Comment:
does this not need the `no_std` flagging? Although, tbh, I dunno if this is really still needed...
```suggestion

// Ensure we're `no_std` when compiling for Wasm.
#![cfg_attr(not(feature = "std"), no_std)]
```

Reply:
This isn't required in the runtime, so we don't need any `no_std` support.

Reply:
supposedly the other feature-flags around `feature = "std"` are also not needed then...?!?

Reply:
Probably  some leftovers from copying the code.

Reply:
@gnunicorn Are there specific `feature = "std"` flags which you think are not needed?

Reply:
well, none in the file are needed if we never build them without `std`. And other stuff in this file implies, it wouldn't compile in `no_std` anyways...

Reply:
Cleaned up as requested